masterlist.yaml

```diff
@@ -3077,6 +3332,61 @@ plugins:
       - Delev
       - Invent
       - Relev
+  - name: 'Lore Weapon Expansion.esp'
+    url: ['https://www.nexusmods.com/skyrimspecialedition/mods/9660/']
+    tag:
+      - Delev
+      - Relev
+    clean:
+      - crc: 0x0D7685E5 # v1.4a
+        util: SSEEdit v3.2.2
+  - name: 'Lore Weapon Expansion - Daedric Crescent.esp'
+    url: ['https://www.nexusmods.com/skyrimspecialedition/mods/9660/']
+    group: *earlyLoadersGroup
+    clean:
+      - crc: 0x524C6138 # v1.0
+        util: SSEEdit v3.2.2
+  - name: 'Lore Weapon Expansion - Goldbrand.esp'
+    url: ['https://www.nexusmods.com/skyrimspecialedition/mods/9660/']
+    group: *earlyLoadersGroup
+    clean:
+      - crc: 0x5587FCE6 # v1.1
+        util: SSEEdit v3.2.2
+  - name: 'Lore Weapon Expansion - Relics of the Crusader.esp'
+    url: ['https://www.nexusmods.com/skyrimspecialedition/mods/9660/']
+    group: *earlyLoadersGroup
+    clean:
+      - crc: 0xA5763C46 # v1.0
+        util: SSEEdit v3.2.2
+  - name: 'Summermyst - Enchantments of Skyrim.esp'
+    url: ['https://www.nexusmods.com/skyrimspecialedition/mods/6285/']
+    inc:
+      - 'Animated Weapon Enchants.esp'
+      - 'Classic Elder Scrolls Attributes.esp'
+      - 'SkyRe_Main.esp'
+      - 'Wintermyst - Enchantments of Skyrim.esp'
+      - 'Worlds Dawn.esp'
+    after:
+      - 'Weapons & Armor Fixes_Remade.esp'
+    tag:
+      - Delev
+      - Graphics
+      - Invent
+      - Relev
+    msg:
+      - <<: *hascompatibilityIssues
+        subs:
+          - 'Summermyst - Enchantments of Skyrim.esp'
+          - 'EnchantingAwakened.esp'
+          - 'Semi-Compatible. Both mods can be active at the same time, but the enchantment limits in Enchanting Awakened do not apply to Summermyst enchantments.'
+        condition: 'active("Summermyst - Enchantments of Skyrim.esp") and active("EnchantingAwakened.esp")'
+      - type: say
+        content:
+          - lang: en
+            text: '[Summermyst Compatibility Notes](https://www.nexusmods.com/skyrimspecialedition/articles/116/?)'
```

Comment:
I think it's useful to add a message link If the compatibility notes are on a separate page or site to the mod description.

Reply:
Yep, I can see that being helpful (I forget articles exist because they're relatively new).